qemu/tests/x86_cpu_test_dies.py

```diff
@@ -0,0 +1,58 @@
+import re
+import random
+
+from avocado.utils import cpu
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_misc
+
+from virttest.utils_version import VersionInterval
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check guest gets correct multiple vcpu dies
+
+    1) Boot guest with options: -smp n,dies=2x...
+    2) Check cpu dies(only for Linux guest and Intel host)
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    qemu_path = utils_misc.get_qemu_binary(params)
+    qemu_version = env_process._get_qemu_version(qemu_path)
+    match = re.search(r'[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?',
+                      qemu_version)
+    host_qemu = match.group(0)
+    # fixed qemu version: qemu-kvm-4.1.0-1.module+el8.1.0+3966+4a23dca1
+    if host_qemu in VersionInterval('[,4.1.0)'):
+        test.cancel("Current qemu version %s doesn't support die." % host_qemu)
```

Comment:
`required_qemu = [4.1.0, )`

Reply:
> `required_qemu = [4.1.0, )`

seems this is equal to current code.....

Reply:
Yes, use parameter to let framework do it.

Reply:
> Yes, use parameter to let framework do it.

Thanks and update.